Squirrel/Resources/SharedResource.uk.resx

```diff
@@ -117,4 +117,32 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Confirm your email" xml:space="preserve">
+    <value>Підтвердіть свою електронну пошту</value>
+    <comment>AccountController.cs</comment>
+  </data>
+  <data name="Invalid login and/or password" xml:space="preserve">
+    <value>Невірний логін та/або пароль</value>
+    <comment>AccountController.cs</comment>
+  </data>
+  <data name="Password reset" xml:space="preserve">
+    <value>Зміна паролю</value>
+    <comment>AccountController.cs</comment>
+  </data>
+  <data name="Passwords are not the same" xml:space="preserve">
+    <value>Паролі не співпадають</value>
+    <comment>AccountController.cs</comment>
+  </data>
+  <data name="Please confirm your account by &lt;a href='{0}'&gt;clicking here&lt;/a&gt;." xml:space="preserve">
+    <value>Будь ласка, підтвердіть свій акаунт &lt;a href='{0}'&gt;натиснувши тут&lt;/a&gt;.</value>
+    <comment>AccountController.cs</comment>
+  </data>
+  <data name="Please confirm your password reset by &lt;a href='{0}'&gt;clicking here&lt;/a&gt;." xml:space="preserve">
+    <value>Будь ласка, підтвердіть зміну паролю &lt;a href='{0}'&gt;натиснувши тут&lt;/a&gt;.</value>
+    <comment>AccountController.cs</comment>
+  </data>
+  <data name="User not found" xml:space="preserve">
```

Comment:
`name` values should fit the string literals in code. It's better to check for possible typos. I found one - the suggestion above